src/test/java/uk/gov/hmcts/darts/arm/service/impl/DetsToArmBatchPushProcessorImplTest.java

```diff
@@ -218,16 +216,16 @@ void processDetsToArmSetObjectStatusNoMatchingDetsRecordErrorMessage() {
     @Test
     void testManifestFileName() throws IOException {
```

Comment:
Since we've touched the test here, consider renaming according to the new naming standard.